runtime/cere/src/lib.rs

```diff
@@ -1612,11 +1656,13 @@ pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, Si
 // 	// migrations::Unreleased,
 // );
 
-// Migrations for Payouts on QANET
-type Migrations = (
-	pallet_ddc_payouts::migrations::v4::MigrateToV4<Runtime>,
-	pallet_ddc_payouts::migrations::v5::MigrateToV5<Runtime>,
-);
+// Migrations for DAC and Payouts on QANET
+
+parameter_types! {
+			pub BalanceTransferAllowDeath: Weight = weights::pallet_balances_balances::WeightInfo::<Runtime>::transfer_allow_death();
+}
+type Migrations =
```

Comment:
Will those migrations be checked by `try-runtime` ?

Reply:
Yes. As a part of CI https://github.com/Cerebellum-Network/blockchain-node/blob/37c3c797c335edc02185c4a444b80e30b915225e/.github/workflows/ci.yaml#L63